rpcs3/Emu/System.h

```diff
@@ -458,3 +479,19 @@ struct cfg_root : cfg::node
 extern cfg_root g_cfg;
 
 extern bool g_use_rtm;
+
+template <>
+inline void fmt_class_string<detail_level>::format(std::string& out, u64 arg)
```

Comment:
Why not have this in system.cpp with all the other enum formatting methods? Its easier to find stuff like this if its all kept in one file